WooCommerce/Classes/ViewRelated/Top Banner/TopBannerView.swift

```diff
@@ -236,6 +239,12 @@ private extension TopBannerView {
             return .warningBackground
         }
     }
+
+    /// Changes the axis of the stack views that need speacial treatment on larger size categories
+    ///
+    func configureStackViewsAxis() {
```

Comment:
super nit: I usually call `update*` if a function might be called more than once after initial setup, to differentiate from `configure*` for one-time initialization. up to you!

Reply:
You are right! :-)